### ISSUE TITLE
Immediately investigate trace from plugin

### ISSUE THREAD

**Issue by rappen** [Innofactor/PluginRegistration#89]
Utilize the new integration possibilities of Plugin Trace Viewer to navigate from Assembly / Plugin / Step directly to PTV with selected context for filtering.

See https://github.com/rappen/PluginTraceViewer/issues/126

---
- Deployment: Online
- DB Version: 9.1.0.25551
- Connection Controls Version: 1.2020.10.35
- XrmToolBox Version: 1.2020.11.42
- Tool Version: 3.2020.11.3

**Comment by imranakram** [comment]
Absolutely, I'd love to link the Auto Deployer and Steps Manipulator as well :)
